src/platform_impl/ios/window.rs

```diff
@@ -224,7 +224,7 @@ impl Inner {
 
     pub fn fullscreen(&self) -> Option<Fullscreen> {
         unsafe {
-            let monitor = self.current_monitor();
+            let monitor = self.current_monitor().unwrap();
```

Comment:
This is mostly a sidenote, but with this way of using the current_monitor API internally we cannot ensure that the typesystem will warn us when things change in the future.

The alternative would be to have an internal `current_monitor` function that returns just `RootMonitorHandle` and then call that in the public `current_monitor` function. That way we can call the internal one here and avoid the `unwrap`.

Reply:
>This is mostly a sidenote, but with this way of using the current_monitor API internally we cannot ensure that the typesystem will warn us when things change in the future.

>The alternative would be to have an internal current_monitor function that returns just RootMonitorHandle and then call that in the public current_monitor function. That way we can call the internal one here and avoid the unwrap.

Do you mean to handle it via `cfg`'s in `src/window.rs`, I guess it could work?

Reply:
No, I mean having an `inner_current_monitor` (or whatever you want to call it) and then have the body of `current_monitor` just be `Some(self.inner_current_monitor())`. So if you just need the monitor internally, you can call `inner_current_monitor` without having to unwrap anything.

Reply:
Ah, gotcha, it makes some sense to do it that way.